client/src/components/NavBar.js

```diff
@@ -13,7 +13,7 @@ function NavBar() {
             <Nav className="me-auto">
             <Nav.Link href="/all-users">All Users</Nav.Link>
             <Nav.Link href="/my-availability">My Availabilities</Nav.Link>
-            <Nav.Link href="/create-availability">Create Availability</Nav.Link>
+            {/* <Nav.Link href="/create-availability">Create Availability</Nav.Link> */}
```

Comment:
If we not going to use this code (which we won't) it's better to remove than leave commented out.

Reply:
Thank you, it's now removed.